python-frontend/src/test/java/org/sonar/plugins/python/api/cfg/ControlFlowGraphTest.java

```diff
@@ -706,6 +707,34 @@ void if_stmt_test_predecessors() {
       "after_if(succ = [END], pred = [before, if_body])");
   }
 
+  @Test
+  void decorators() {
+    ControlFlowGraph cfg = cfg("" +
+      "class Dec:",
```

Comment:
I don't quite understand why we prepend the first element with the empty string. Is this intentional?